packages/js/settings-editor/src/hooks/utils/transformers.tsx

```diff
@@ -11,12 +11,25 @@ import type { Field, FormField } from '@wordpress/dataviews';
  */
 import { CustomView } from '../../components/custom-view';
 import { SettingsGroup } from '../../components/settings-group';
-import { CheckboxEdit } from '../../components/checkbox-edit';
+import { getCheckboxEdit } from '../../components/checkbox-edit';
 import { getInputEdit } from '../../components/inputEdit';
-import { SelectEdit } from '../../components/selectEdit';
+import { getSelectEdit } from '../../components/selectEdit';
 
 export type DataItem = Record< string, BaseSettingsField[ 'value' ] >;
 
+/**
+ * Helper function to determine label and help text from setting
+ * If desc_tip is true, we should display the desc as the help text.
+ */
+const getLabelAndHelp = (
```

Comment:
👍🏽 For reusable function - text areas and surely a few others will make use as well